islandora_pdfjs.module

```diff
@@ -78,3 +78,95 @@ function islandora_pdfjs_viewer_callback(array $params, $fedora_object = NULL) {
     return theme('islandora_pdfjs', array('fedora_object' => $fedora_object, 'dsid' => $dsid));
   }
 }
+
+/**
+ * Implements hook_CMODEL_PID_islandora_solr_object_result_alter().
+ */
+function islandora_pdfjs_islandora_bookCModel_islandora_solr_object_result_alter(&$search_results, $query_processor) {
+  module_load_include('inc', 'islandora', 'includes/solution_packs');
+  $viewer_id = islandora_get_viewer_id('islandora_book_viewers');
+  if ($viewer_id == 'islandora_pdfjs') {
+    islandora_pdfjs_add_solr_search_param($search_results, $query_processor);
+  }
+}
+
+/**
+ * Implements hook_CMODEL_PID_islandora_solr_object_result_alter().
+ */
+function islandora_pdfjs_islandora_sp_pdf_islandora_solr_object_result_alter(&$search_results, $query_processor) {
+  module_load_include('inc', 'islandora', 'includes/solution_packs');
+  $viewer_id = islandora_get_viewer_id('islandora_pdf_viewers');
+  if ($viewer_id == 'islandora_pdfjs') {
+    islandora_pdfjs_add_solr_search_param($search_results, $query_processor);
+  }
+}
+
+/**
+ * Implements hook_CMODEL_PID_islandora_solr_object_result_alter().
+ */
+function islandora_pdfjs_islandora_newspaperIssueCModel_islandora_solr_object_result_alter(&$search_results, $query_processor) {
+  module_load_include('inc', 'islandora', 'includes/solution_packs');
+  $viewer_id = islandora_get_viewer_id('islandora_newspaper_issue_viewers');
+  if ($viewer_id != 'islandora_pdfjs') {
```

Comment:
Should you check for equality or inequality here? Above you used `==`, here you use `!=`.

Reply:
@nhart @bencomp makes a valid point here. Should the viewer_id here be 'islandora_pdfjs' to trigger  islandora_pdfjs_add_solr_search_param?

Reply:
Looks like a copy and paste error.